adapters/zwave/zwave-classifier.js

```diff
@@ -46,7 +47,28 @@ class ZWaveClassifier {
             'name': 'toggle',
             'description': 'Toggles On/Off',
         }];
-        node.propertyMap = {'on': valueId };
+        node.propertyMaps = {'on': valueId };
+        node.values.on = node.zwValues[valueId].value;
+
+        if (node.zwInfo.productId == 0x0060 &&
+            (node.zwInfo.productType & 0xff) == 0x03) {
```

Comment:
After implementing this a few times (I now have zigbee, zwave, and gpio onOffThings, I think that we need to factor some of this out, but I'll leave that for later.